dev/com.ibm.ws.app.manager.war/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -31,4 +31,9 @@ classloader.desc=Defines the settings for the application classloader.
 
 appBnd.name=Application Bindings
 appBnd.name$Ref=Application bindings reference
-appBnd.desc=Binds general deployment information included in the application to specific resources.
\ No newline at end of file
+appBnd.desc=Binds general deployment information included in the application to specific resources.
+
+earapp.addWebModuleClassPathTo.name=The class loader used for Web Application class paths
+earapp.addWebModuleClassPathTo.desc=Defines the class loader used for JARs referenced by Web Application Class-Path headers.
+earapp.addWebModuleClassPathTo.warLoader=Use the Web Application class loader for the JARs referenced by a Web Application Class-Path header.
```

Comment:
```suggestion
earapp.addWebModuleClassPathTo.warLoader=Use the web application class loader for the JARs that are referenced by a web application class-path header.
```

is this a description of this attribute? Does it take a true/false value or specify a particular class loader? Im not clear on what we want to tell the user about this attribiute

Reply:
ok- generally we are asked not to capitalize anything except for stand-alone licensed products. But if `Class-Path` is how it appears in code, thats the way it should appear here.

Thanks for the explanation - makes sense. I wonder if it would be helpful to add something like

> Defines the class loader that is used for JARs that are referenced by web application Class-Path headers. Specify one of the following values....

but im not sure if this description might be used in other contexts where the value descriptions dont appear just after, ie does it need to be able to make sense without them

Reply:
I don't think the attribute description is shown without the options and the option descriptions.